src/components/UI/Button/button.story.tsx

```diff
@@ -9,51 +10,87 @@ const meta: Meta<typeof Button> = {
     layout: 'centered',
   },
   tags: ['autodocs'],
+  args: {
+    onClick: fn(),
```

Comment:
Why is this part needed? we already checking the click function in the test function no ?

Reply:
The check requires a mock function to work. It does check the mock function but it's still required because there would be no object to check the call count.